packages/client/e2e/credit.spec.ts

```diff
@@ -148,7 +148,7 @@ test.describe("Credit", () => {
 
       await sellSidePage.waitForSelector("text=New RFQ")
 
-      await sellSidePage.getByTestId("price-input").fill("100")
+      await sellSidePage.getByTestId("price-input").pressSequentially("100")
```

Comment:
nice - are there any more lying around?
in some cases a `.fill` will be appropriate, but maybe as a parallel test, to check what happens if people paste values in - can't think of one off-hand in RT, mind you .. RA perhaps?
(I am referring to the Playwright 1.38 update video)

Reply:
There is no explicit test validating how pasting a value in a text field behave, either in RT or RA. There are still few `.fill` being used in RT.  I will update them to `.pressSequentially` in a subsequent PR